packages/apps/docs/src/pages/docs/marmalade/example-policies/index.mdx

```diff
@@ -0,0 +1,8 @@
+---
+title: Example Policies
+description: Example Policies
+menu: Example Policies
+label: Example Policies
+order: 5
+layout: normal
+---
```

Comment:
this page doesn't have any actual content content at the moment. Do we actually need this? 
<img width="1619" alt="image" src="https://github.com/kadena-community/kadena.js/assets/7163943/4541d26d-0a31-4838-a019-8ab05f3dfc97">